src/appleseed/renderer/modeling/object/meshobject.cpp

```diff
@@ -37,6 +37,7 @@
 
```

Comment:
The downside is that we already have duplicated handling of alpha maps in `Object` and `MeshObject`. We could have *more* functionality, more orthogonality, and *less* code if everything was just in `Object`. But again, your call.